pkg/asset-manager-utils/contracts/AssetManager.sol

```diff
@@ -65,8 +67,16 @@ abstract contract AssetManager is IAssetManager {
      * @return The amount of the underlying tokens which are owned by the specified pool
      */
     function balanceOf(bytes32 poolId) public view returns (uint256) {
+        return _poolManaged(poolId, readAUM());
+    }
+
+    /**
+     * @param poolId - The id of the pool of interest
+     * @return The amount of the underlying tokens which are owned by the specified pool
+     */
+    function _poolManaged(bytes32 poolId, uint256 aum) public view returns (uint256) {
```

Comment:
This puts a leading underscore in the interface, which we probably don't want

Reply:
Perhaps this function was meant to be internal?

Reply:
Whoops, this is accidental. I've just made this internal.